tests/cms/utils/test_slug_utils.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING
+
+import pytest
+from django.core.exceptions import ValidationError
+from pytest_django.fixtures import SettingsWrapper
+
+from integreat_cms.cms.models.pages.page_translation import PageTranslation
+from integreat_cms.cms.models.regions.region import Region
+from integreat_cms.cms.utils.slug_utils import generate_unique_slug
+
+if TYPE_CHECKING:
+    from integreat_cms.cms.utils.slug_utils import SlugKwargs
+
+
+@pytest.mark.django_db
+def test_generate_unique_slug_fallback(
+    settings: SettingsWrapper, load_test_data: None
+) -> None:
+    """
+    Test whether the :meth:`~integreat_cms.cms.utils.slug_utils.generate_unique_slug_helper` function correctly uses the fallback property
+    when no slug is provided
+    """
+    settings.DEBUG = True
+    mock_object = Region.objects.get(slug="augsburg")
+    print(mock_object)
+    kwargs: SlugKwargs = {
+        "cleaned_data": {"name": "unique_slug_fallback"},
+        "manager": Region.objects,
+        "object_instance": mock_object,
+        "foreign_model": "region",
+        "fallback": "name",
+    }
+    assert generate_unique_slug(**kwargs), "unique_slug_fallback"
```

Comment:
```suggestion
    assert (
        generate_unique_slug(**kwargs) == "unique_slug_fallback"
    ), "Name is not used as fallback when slug is missing"
```
The syntax of the [assert](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement) statement only evaluates the first expression, the second one after the `,` is an optional message that is printed when the assertion fails. So `assert generate_unique_slug(**kwargs), "unique_slug_fallback"` only checks whether `generate_unique_slug(**kwargs)` evaluates to something "truthy" and would print the literal "unique_slug_fallback" when the result is falsey. This is also the reason why the other two tests are passing even though the if condition is incorrect.

Reply:
Uhhm. Oops :D. 🧑‍🏭